013-quarkus-oidc-restclient/pom.xml

```diff
@@ -26,6 +30,15 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-rest-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-openapi</artifactId>
+        </dependency>
+<!--  Looks that it's not required, however Docs says the opposite. https://quarkus.io/guides/rest-client#async-support        -->
```

Comment:
I think it is thanks to `quarkus-resteasy-mutiny` dependency on line 17 + `quarkus-rest-client` on line 29.

the missing part is probably just `extensions/rest-client-mutiny/deployment/src/main/java/io/quarkus/restclient/mutiny/deployment/RestClientMutinyProcessor.java`

I think `quarkus-rest-client` could be removed and `quarkus-rest-client-mutiny` enabled to be safe for the future (if rest-client-mutiny gets enhanced)

Reply:
https://mvnrepository.com/artifact/io.quarkus/quarkus-rest-client-mutiny/1.11.0.Final